src/components/Favourite.js

```diff
@@ -0,0 +1,103 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { BsArrowLeft, BsArrowRight } from 'react-icons/bs';
+import Navbar from './navbar';
+import { fetchfavourites } from '../redux/favouritesSlice';
+import StarRating from './StarRating';
+import '../styles/main.css';
+
+function Favourite() {
```

Comment:
- [ ] If I reserve a car and immediately go to the "Favorites" link, this happens:

![image](https://github.com/Rhaegar121/Capstone-Front-End/assets/57421823/bc30b3a1-cbeb-4265-9597-232e97e879c4)

This seems to be caused because the favorites list has changed, and the server has to fetch the cars list again.

Kindly fix this bug so your app works as intended. 👍